lib/git-lfs-s3/services/ceph_presigner.rb

```diff
@@ -0,0 +1,24 @@
+require 'base64'
+require 'date'
+require 'digest/sha1'
+require 'openssl'
+require 'uri'
+
+module GitLfsS3
+  module CephPresignerService
+    extend self
+    extend AwsHelpers
+
+    def signed_url(obj)
+      expire_at = (DateTime.now + 1).strftime("%s")
+      secret_access_key = Aws.config[:secret_access_key]
+      access_key_id = Aws.config[:access_key_id]
```

Comment:
I added a new environment variable. LFS_CEPH_ENDPOINT since it'll be required to configure this server to use a Ceph S3 service. This was an oversight - I had it hardcoded in our implementation and it should be explicitly configurable.